include/net/lwm2m.h

```diff
@@ -1026,5 +1032,15 @@ void lwm2m_rd_client_start(struct lwm2m_ctx *client_ctx, const char *ep_name,
 void lwm2m_rd_client_stop(struct lwm2m_ctx *client_ctx,
 			  lwm2m_ctx_event_cb_t event_cb);
 
+			  /**
+ * @brief Trigger a Registration Update of the LwM2M RD Client
+ *
+ * The RD client sits just above the LwM2M engine and performs the necessary
```

Comment:
Actually, I think the brief description is enough, no need to carry this comment around the functions.